crates/bevy_render/src/primitives/mod.rs

```diff
@@ -241,18 +241,24 @@ impl Frustum {
 
     /// Returns a frustum derived from `clip_from_world`,
     /// but with a custom far plane.
+    /// If `far` is `None` behavior is identical to [`Frustum::from_clip_from_world`]
```

Comment:
This seems like the least expectation defying change but not sure it's the best approach?

Reply:
Changed.